composer.json

```diff
@@ -14,7 +14,8 @@
         }
     },
     "require": {
-        "amphp/artax": "^3",
+        "php": "^7.0",
```

Comment:
But we do not support PHP8. 

If one should be super strict we should write: `7.0.* || 7.1.*`. Do you think we should do that?